cluster/repository/cluster_repository.go

```diff
@@ -108,13 +108,13 @@ func (m *GormClusterRepository) Load(ctx context.Context, id uuid.UUID) (*Cluste
 	return &native, errs.WithStack(err)
 }
 
-// LoadClusterByURL returns a single Cluster filtered using 'url'
-func (m *GormClusterRepository) LoadClusterByURL(ctx context.Context, url string) (*Cluster, error) {
+// FindByURL returns a single Cluster given its 'url'
+func (m *GormClusterRepository) FindByURL(ctx context.Context, url string) (*Cluster, error) {
 	defer goa.MeasureSince([]string{"goa", "db", "cluster", "loadClusterByURL"}, time.Now())
 	var native Cluster
 	err := m.db.Table(m.TableName()).Where("url = ?", url).Find(&native).Error
```

Comment:
As discussed in https://github.com/fabric8-services/fabric8-cluster/issues/66 I think we should normalize `url` here. If we always store cluster URLs with trailing slashes in DB then let's add a trailing slash to the `url` param here too if it doesn't already have it.

Reply:
ok, let me address #66 first then, and then I'll revisit this PR ;)

Reply:
URL normalization was addressed in #66 (including when [searching by URL](https://github.com/fabric8-services/fabric8-cluster/blob/master/cluster/repository/cluster_repository.go#L162)) and merged in this PR in commit 7f914e2